src/components/MainBanner/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import { IncrementPercentage } from '../IncrementPercentage'
+import { DataCard } from  '../DataCard'
+
+import './MainBanner.scss'
+import mexicoImage from '../../assets/mexico.jpg'
+
+export const MainBanner = () => {
```

Comment:
Should be default export, if want to share more components it's ok but you  are only sharing 1 and always need a default

Reply:
get it, I will change it